app/off.py

```diff
@@ -11,20 +11,36 @@
 from .wikidata_utils import get_wikidata_entity, image_thumbnail, wikidata_props
 
 
-@alru_cache()
-async def global_quality(source_url):
+async def fetch_quality(source_url):
     async with aiohttp.ClientSession() as session:
         quality_url = f"{source_url}/data-quality.json"
         async with session.get(quality_url) as resp:
             return await resp.json()
 
 
+# cached version of fetch_quality for slow requests
+global_quality = alru_cache(fetch_quality)
+
+DataQuality = namedtuple(
+    "DataQuality",
+    [
+        "text",
+        "source_url",
+        "description",
+        "title",
+    ],
+)
+
+
 async def data_quality(url, path):
     """
     Helper function to return issues for data-quality
     """
     source_url = urljoin(url, path)
-    data = await global_quality(source_url)
+    try:
+        data = global_quality if path == "" else await fetch_quality(source_url)
```

Comment:
you have to call global_quality, this is a function, with same signature as fetch_quality (a decorator applied to a function returns a function)

Reply:
```suggestion
        data = global_quality(source_url) if path == "" else await fetch_quality(source_url)
```